### DISCUSSION TITLE
Make it easier to find the setting to turn off dependabot

### DISCUSSION THREAD

**Discussion by StingyJack** [community/community#19233] [Code Security]
Dependabot is triggering notifications for some of my repos. The repos it reports are only being kept around so I can refer to it from time to time, so I want to turn off dependabot for them. 

The setting is kind of buried in an inconsistent navigation experience and should be right on the Security tab

**Comment by felicitymay** [comment]
Maybe one of these will help:
- [Configuring Dependabot alerts](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/configuring-dependabot-alerts#managing-dependabot-alerts-for-your-repository)
- [Disable vulnerability alerts](https://docs.github.com/en/rest/repos/repos#disable-vulnerability-alerts)
